README.md

```diff
@@ -33,3 +35,14 @@ To know about the available options run the command:
 To test changes, enter the `tests` directory and run the following command:
 
     pytest tests.py
+    
+    
+## Output
```

Comment:
Can we "Output" to something like "Example plots"? What do you think?

Reply:
Sure. What the program will give as output is already mentioned in the description